third_party/amd/lib/TritonAMDGPUTransforms/AccelerateAMDMatmul.cpp

```diff
@@ -1008,11 +1254,17 @@ class TritonAMDGPUAccelerateMatmulPass
     switch (auto isaFamily = triton::AMD::deduceISAFamily(archGenerationName)) {
     case ISAFamily::CDNA1:
     case ISAFamily::CDNA2:
-    case ISAFamily::CDNA3:
+    case ISAFamily::CDNA3: {
+      patterns.add<::ScaledBlockedToScaledMFMAF8F6F4>(
+          context, getMfmaVersion(isaFamily, archGenerationName),
+          matrixInstructionSize,
+          /*benefit=*/10);
```

Comment:
Style nit: These two lines can be concatenated together.

Reply:
Oh didn't realize the formatter won't help me with that. 🥲